### PR TITLE
Documentation/v1.0.0 rc1 doc update

### PR DESCRIPTION
**Pull request by sampsapenna**
### Description

Documentation is overdue an upgrade – PR adds some information that is necessary for the building and use of the more complete v1.0.0rc1.

### Related issues

* #54 – Update documentation

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (this needs a follow up PR)

### Changes Made

* Write deployment documentation for the additional microservices
* Update UI documentation on the additional features
* Update screenshots to represent the current version

### Testing

- [ ] Unit Tests
- [ ] Integration Tests
- [x] Tests do not apply
- [ ] Needs testing (start an issue or do a follow up PR about it)

### CODE REVIEW

File: docs/source/deploy.rst [hunk 1]
```diff
@@ -21,12 +21,88 @@ Using vanilla docker in order to build the image - the tag can be customised:
                  -e BROWSER_START_AUTH_ENDPOINT_URL=https://pouta.csc.fi:5001/v3 \
                  cscfi/swift-ui
 
+Database for sharing functionality
+----------------------------------
+Both ``swift-x-account-sharing`` and ``swift-sharing-request`` services need
+access to a Postgres database in order to work. In the usual deployment this
```
[the remark this answers is not in the document]
Comment (sampsapenna) [reply docs/source/deploy.rst]: Added

File: docs/source/deploy.rst [hunk 2]
```diff
@@ -21,12 +21,88 @@ Using vanilla docker in order to build the image - the tag can be customised:
...
+is done with a container deployment. Necessary files to build a database
```
[the remark this answers is not in the document]
Comment (sampsapenna) [reply docs/source/deploy.rst]: Fixed

File: docs/source/index.rst [hunk 3]
```diff
@@ -24,14 +25,17 @@ Out of the box the ``swift-browser-ui`` offers:
...
-    User Interface           <ui>
-    Python Modules           <code>
-    Testing                  <testing>
-    Tools / Miscellaneous    <tools>
+    Setup Instructions         <instructions>
+    Usage & Examples           <usage>
+    Deployment                 <deploy>
+    Architecture               <tech>
+    User Interface             <ui>
+    Sharing backend            <sharing>
```
[the remark this answers is not in the document]
Comment (sampsapenna) [reply docs/source/index.rst]: Fixed on all rows

File: docs/source/runner.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+
+File Upload
+-----------
+Files can be uploaded to an automatically generated container by drag'n'drop
+from the container listing page, or by using the upload button on top of the
+table. There's no limit on how large the files uploaded can be, but browser
+performance puts a practical limit somewhere in the neighborhood of 10GiB.
+
+Uploading files to a specific container can be done by opening the container,
```
[the remark this answers is not in the document]
Comment (sampsapenna) [reply docs/source/runner.rst:16]: Emphasized that the page talks about object storage container operations in the first paragraph, is this enough?

File: docs/source/sharing.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    :align: center
+
+    Image of the view listing containers shared from the project
+
+Viewing containers shared to the project
+----------------------------------------
+Containers that are shared to a particular project can be viewed by navigating
+to the "Shared" page in the application navbar. From this view the granted
+access can be viewd, and any container can be opened just like when using
+the normal container browsing view. All features present in the ordinary
```
[the remark this answers is not in the document]
Comment (sampsapenna) [reply docs/source/sharing.rst]: Yup, that's a better alternative, changed

File: docs/source/instructions.rst [hunk 6]
```diff
@@ -10,8 +10,19 @@ The program can be installed with pip from the git repository:
 
     # Requires python >= 3.6
     git clone git@github.com:CSCfi/swift-browser-ui.git
+    # Frontend files need to be separately built
+    cd swift_browser_ui_frontend && npm run build && cd ..
     pip install .
 
+Note that the program uses external services that need to be present in order
```
[the remark this answers is not in the document]
Comment (sampsapenna) [reply docs/source/instructions.rst]: Yup, a note makes sense, changed

File: docs/source/instructions.rst [hunk 7]
```diff
@@ -10,8 +10,19 @@ The program can be installed with pip from the git repository:
...
+to enable all functionality, like sharing. These additional services can be
+found from the git repositories. The instructions to getting the services up
```
[the remark this answers is not in the document]
Comment (sampsapenna) [reply docs/source/instructions.rst]: Yup, for is correct, changed

File: docs/source/runner.rst [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+project that is currently active. To the user the download continues to be
+a simple download link
+
+
+Container Download
+------------------
+Full containers can be downloaded from the UI using the download button either
+on the table row in the container listing, or a download button on the top of
+the table when viewing an open container. Downloading whole containers works
+the same in both owned and shared containers. The runner seamlessly creates
```
[the remark this answers is not in the document]
Comment (sampsapenna) [reply docs/source/runner.rst]: Improved the phrasing a bit

File: docs/source/runner.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (blankdots) [reply docs/source/runner.rst:16]: sure

**Pull request merged**